mathml/relations/css-styling/padding-border-margin/border-002.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>border</title>
+<link rel="help" href="https://mathml-refresh.github.io/mathml-core/#layout-algorithms">
+<meta name="assert" content="Verify that border is taken into account.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/mathml/support/mathml-fragments.js"></script>
+<script src="helper.js"></script>
+<script>
+  var epsilon = 1;
+
+  setup({ explicit_done: true });
+  window.addEventListener("load", runTests);
+
+  function runTests() {
+
+    for (tag in MathMLFragments) {
+        if (!FragmentHelper.isValidChildOfMrow(tag))
+            continue;
+
+        var style = "border-left: 30px solid transparent; border-right: 40px solid transparent; border-top: solid transparent 50px; border-bottom: 60px solid transparent;";
```

Comment:
It may be worth trying border: solid transparent to make this shorter.

Reply:
Something like

    <div style="border: solid black; border-left: 1px;">

does not seem to have border left.

Reply:
Ah, border-left always needs a style, so my suggestion only saves transparent, up to you if you try that.

Reply:
OK, I'll actually just remove "transparent" as that's not really needed.